main.go

```diff
@@ -264,7 +264,7 @@ func setupProbeEndpoints(mgr ctrl.Manager) {
 	}
 }
 
-func setupScheduler(mgr ctrl.Manager, cCache *cache.Cache, queues *queue.Manager, cfg *configapi.Configuration) {
+func setupScheduler(mgr ctrl.Manager, cCache *cache.Cache, queues *queue.Manager) {
```

Comment:
@mimowo wasn't this required for WaitForPodsReady at some point?

Although I think we changed the logic to only be checked in the queue manager, correct?

Reply:
Indeed, it was a leftover after the refactoring.